src/main/java/org/triplea/ui/TripleaMainMenu.java

```diff
@@ -0,0 +1,174 @@
+package org.triplea.ui;
+
+import java.awt.Desktop;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.text.MessageFormat;
+
+import games.strategy.debug.ClientLogger;
+import games.strategy.engine.ClientContext;
+import games.strategy.triplea.UrlConstants;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.control.Button;
+import javafx.scene.control.PasswordField;
+import javafx.scene.control.TextField;
+import javafx.scene.input.MouseEvent;
+import javafx.scene.layout.BorderPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.scene.text.Text;
+
+public class TripleaMainMenu extends BorderPane {
+
+  private final TripleA triplea;
+  private final TripleaDownload downloadPane;
+  private final TripleaSettings settingsPane;
```

Comment:
Since the variable names end with `Pane`, does that suggest the class names should, as well?  I'll admit I was a little confused by the names `TripleaDownload` and `TripleaSettings`.  I first asked myself, "why is @RoiEXLab creating non-UI classes for a 'download' and 'settings' in this package."  Then I realized they were the panes responsible for displaying that information in the UI. :smile: